src/backend/px-manager.c

```diff
@@ -669,6 +669,10 @@ px_manager_get_proxies_sync (PxManager  *self,
 
     g_debug ("%s: Config[%d] = %s", __FUNCTION__, idx, config[idx]);
 
+    /* Ignore invalid proxy URL, so we won't call GUri API on NULL. */
+    if (conf_url == NULL)
```

Comment:
It's correct to add such a check, but we tend to use the other notation

`if (!cont_url)`

Could you please update your PR?

Reply:
OK! Please wait.

Reply:
Done.